tests/send_CAPI_command.py

```diff
@@ -15,6 +15,54 @@ class CAPIReply(Enum):
     INVALID = "status,INVALID"
     ERROR = "status,ERROR"
 
+class tlv:
+    """Representation of an 1905.1 TLV."""
+
+    def __init__(self, type_: int, length: int, value: str):
+        """A TLV has a type, a length and a value.
+
+        Parameters
+        ----------
+        type_ : int
+            The TLV type.
+
+        length : int
+            The length of the TLV. This must correspond to the length of `value`.
+
+        value : str
+            The TLV value, as a string. It must be formatted according to the UCC rules, i.e. with
+            curly braces and hexadecimal numbers.
+        """
+        self.type = type_
+        self.length = length
+        self.value = value
+
+    def format(self, tlv_num: int = 0) -> str:
```

Comment:
for ease of debugging, it could be useful to rename this function `__str__` (or `__repr__`), so you can just to `print(tlv)`.

Reply:
This format function does something different than simply make a string representation, so NACK. I could add a __str__ that prints the three members though.